hexa/databases/tests/test_api.py

```diff
@@ -0,0 +1,26 @@
+import re
+
+from django.core.exceptions import ValidationError
+
+from hexa.core.test import TestCase
+from hexa.databases.api import create_database, format_db_name
+
+
+class DatabaseAPITest(TestCase):
+    def test_format_db_name(self):
+        db_name = "RDC_POLIO_PROJECT"
+        self.assertEqual(len(db_name), len(format_db_name(db_name)))
```

Comment:
Better to compare the result of format_db_name than the length of the result. You might end up with the same number of characters but completely different from what you'd like to have.